test/testinput/bufr_sfcshp.yaml

```diff
@@ -0,0 +1,101 @@
+# (C) Copyright 2020 NOAA/NWS/NCEP/EMC
+#
+# This software is licensed under the terms of the Apache Licence Version 2.0
+# which can be obtained at http://www.apache.org/licenses/LICENSE-2.0.
+
+observations:
+  - obs space:
+      name: bufr
+      #obsdatain: "./testdata/bufr_sfcshp.bufr"
+      #obsdatain: "./testdata/shipsu.20180415_subsampled"
+      obsdatain: "./testdata/ships.20180415_subsampled"
+
+      exports:
+        group_by_variable: depth
+        variables:
+          timestamp:
+            datetime:
+              year: "*/YEAR"
+              month: "*/MNTH"
+              day: "*/DAYS"
+              hour: "*/HOUR"
+              minute: "*/MINU"
+              second: ""
+
+          stationID:
+            query: "*/RPID"
+
+          longitude:
+            query: "*/CLONH"
+
+          latitude:
+            query: "*/CLATH"
+
+          sst:
+            query: "*/SST0"
+
```

Comment:
to convert from K to degC add:
            transforms:
              - offset: -273.15
  
below: query: "*/SST0"

Reply:
Please query MSST (This is the method of SST measurement and is important for adding ObsError values. On ship, SST is measured using 2/3 locations, e.g ERI (Engine Roome Intake), Hull, bucket)

Reply:
Added MSST, but the results in ncdump are as follows:

>    method = 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 
>       0, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 
>       0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 
>       0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0 ;

Also, as I get a better idea how what to look for, SST0 appears in the bufr_dump but the ncdump of the result is blank. SST1 has values; is this what is needed?